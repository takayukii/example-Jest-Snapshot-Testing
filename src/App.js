import React from 'react';

function App(props = {}) {
  return (
    <div className="container">
      <div className="jumbotron">
        <h1>{props.greeting}</h1>
      </div>
    </div>
  )
}

export default App
