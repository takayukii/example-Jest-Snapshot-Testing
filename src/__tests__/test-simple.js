import React from 'react';
import renderer from 'react-test-renderer';

import App from '../App';

describe('<App/>', () => {
  it('snapshot1', () => {
    const component = renderer.create(
      <App greeting="Hello simple"/>
    );
    expect(component).toMatchSnapshot('Greeting Hello simple is displayed?');
  });
});
