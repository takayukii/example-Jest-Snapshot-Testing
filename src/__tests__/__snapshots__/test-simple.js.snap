// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting Hello simple is displayed? 1`] = `
<div
  className="container"
>
  <div
    className="jumbotron"
  >
    <h1>
      Hello simple
    </h1>
  </div>
</div>
`;
